packages/nx-plugin/package.json

```diff
@@ -6,15 +6,15 @@
   "generators": "./generators.json",
   "executors": "./executors.json",
   "dependencies": {
-    "@nx/devkit": "^20.2.2",
+    "@nx/devkit": "^20",
```

Comment:
simplified dep resolution to mitigate churn when migrating to new nx versions in repo